apps/web/src/app/(signing)/sign/[token]/date-field.tsx

```diff
@@ -82,13 +82,17 @@ export const DateField = ({ field, recipient }: DateFieldProps) => {
         </div>
       )}
 
-      {!field.inserted && (
-        <p className="group-hover:text-primary text-muted-foreground text-lg duration-200">Date</p>
-      )}
-
-      {field.inserted && (
-        <p className="text-muted-foreground text-sm duration-200">{field.customText}</p>
-      )}
+      <p
```

Comment:
With this one I opted to not use ternaries since it's less to keep in mind while reading the component. With these changes you have to keep both ternaries in mind which can make debugging more annoying 😇

Reply:
Gotcha, that makes sense. I will revert this change then :)

Reply:
Done 🚀